src/app/GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -221,9 +221,15 @@ private void EnableRaisingEvents()
                 return;
             }
 
-            _workTreeWatcher.EnableRaisingEvents = Directory.Exists(_workTreeWatcher.Path);
-            _gitDirWatcher.EnableRaisingEvents = Directory.Exists(_gitDirWatcher.Path)
-                    && !_gitDirWatcher.Path.StartsWith(_workTreeWatcher.Path);
+            try
+            {
+                _workTreeWatcher.EnableRaisingEvents = Directory.Exists(_workTreeWatcher.Path);
+                _gitDirWatcher.EnableRaisingEvents = Directory.Exists(_gitDirWatcher.Path)
+                        && !_gitDirWatcher.Path.StartsWith(_workTreeWatcher.Path);
+            }
+            catch
+            {
```

Comment:
```suggestion
            catch
            {
                _workTreeWatcher.EnableRaisingEvents = false;
                _gitDirWatcher.EnableRaisingEvents = false;
```

Reply:
It is OK to watch worktree also if gitdir is not possible